src/pages/_app.js

```diff
@@ -166,3 +166,13 @@ function MyApp({ Component, pageProps }) {
 }
 
 export default appWithTranslation(MyApp);
+
+MyApp.getInitialProps = async ({ Component, ctx }) => {
+    let pageProps = {};
+
+    if (Component.getInitialProps) {
+        pageProps = await Component.getInitialProps(ctx);
+    }
+
+    return { pageProps, namespacesRequired: ["common"] };
+};
```

Comment:
I guess this is the important piece of puzzle. Since our layout component is included in every page, adding a `namespacesRequired` seems to remove the warnings. I haven't updated the next-i18next version. That could be handled in separate PR.